### DISCUSSION TITLE
Error message: No FileSystem for scheme "s3"

### DISCUSSION THREAD

**Discussion by as10128** [apache/amoro#2479] [General]
I using hadoop catalog, when viewing the table properties, the following error message is returned: Caused by: org.apache.hadoop.fs.UnsupportedFileSystemException: No FileSystem for scheme "s3"
<img width="1322" alt="image" src="https://github.com/NetEase/amoro/assets/22188671/b067f1d7-7e4f-4b12-95fa-8f93d3795412">

<img width="992" alt="image" src="https://github.com/NetEase/amoro/assets/22188671/0082a7af-476d-4b69-b6a8-a7c6120d78e9">

**Comment by baiyangtx** [comment]
What's the value of `fs.defaultFS` in your core-site.xml ?

**Reply by as10128** [reply]
<img width="552" alt="image" src="https://github.com/NetEase/amoro/assets/22188671/5907124a-2a33-47ea-9ebc-27a631b25959">

**Comment by baiyangtx** [comment]
Additionally, I must remind you that when using `Hadoop Catalog`, there may be atomic commit risks when Iceberg is combined with the S3 storage system. It is recommended to use the `Hive Catalog` or `InternalCatalog`.

**Reply by as10128** [reply]
Thanks for the reminder.
